ci/long_running_tests/config.yaml

```diff
@@ -14,48 +14,35 @@ auth:
 
 head_node:
     InstanceType: m5.xlarge
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
 worker_nodes:
     InstanceType: m5.large
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
     # Run workers on spot by default. Comment this out to use on-demand.
     InstanceMarketOptions:
         MarketType: spot
 
 # List of shell commands to run to set up nodes.
 setup_commands:
-    - sudo apt-get update
-    - sudo apt-get install -y build-essential curl unzip
-    # Install Anaconda.
-    - wget https://repo.continuum.io/archive/Anaconda3-5.0.1-Linux-x86_64.sh || true
-    - bash Anaconda3-5.0.1-Linux-x86_64.sh -b -p $HOME/anaconda3 || true
-    - echo 'export PATH="$HOME/anaconda3/bin:$PATH"' >> ~/.bashrc
-    - echo 'termcapinfo xterm* ti@:te@' >> ~/.screenrc
-    # Some Python dependencies.
-    - pip install boto3==1.4.8 cython==0.29.0
-    # Uncomment the following if you wish to install Ray instead.
+    # Install nightly Ray wheels.
+    - source activate tensorflow_p36 && pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev1-cp36-cp36m-manylinux1_x86_64.whl
```

Comment:
`dev2`?

Reply:
Also can you keep the `https://s3-us-west-2.amazonaws.com/ray-wheels/<<<RAY_BRANCH>>>/<<<RAY_COMMIT>>>/ray-<<<RAY_VERSION>>>-cp36-cp36m-manylinux1_x86_64.whl`? Specifying the commit and ray version is nice...

Reply:
Yes, let's leave it the way that it was before so that it forces the user to specify the branch/version/commit. I added that because it basically guarantees that the caller will be testing the thing they intend to test (whereas before it was very easy to test the wrong thing).